infra/license/README.md

```diff
@@ -0,0 +1,54 @@
+# License Self Verification
+
+## Background
+
+Numerous kinds of Node packages are needed to build and use ONE-vscode but some packages may have unacceptable license for both ONE and ONE-vscode.
+Therefore we should review whether licenses of each package can be used or not.
+However, if we check all of them manually, we may mistake or miss something.
+To reduce such risk and labor, automatic license checker would be good solution.
+
+## Process & Architecture
+
+### Flowchart
+
+![flowchart](flowchart.png)
+
+### Terminology
+
+- license-judgment.json
+  - Each of license names are got from the result of `license-checker`.
+  - Each license are classfied to following categories
+    - `Allowed` : Allowed to use
+    - `Warning` : Manual review for each package is needed
+    - `Denied` : Not allowed to use
+- package-judgment.json
+  - Each judgment for a package is consisted of following information
+    ```json
+    "pkgname@version": {
+      "licenses": "License name",
+      "repository": "URL for package repository",
+      "ONE_permitted": "yes / conditional / no",
```

Comment:
Can I ask a somewhat basic question? 
If a package `foo` has license 'MIT`, we can know we can use `foo` by looking at `license-judgment.json`.
So, I wonder the reason of having `package-judgment.json` as a separate file.

Reply:
If all packages are only consisted of such cases, `package-judgment.json` would not be needed :)
The file is need for following cases

- If license is `UNKNOWN`, how can we know the license?
- If license is classified as `Warning`, can we judge whether it can be used or not by only license name?
  - e.g. `(MIT OR CC0-1.0)`
- If package cannot be used but has `Allowed` license, how can we forbid the package?

Because of these reason, I created `package-judgment.json` and this file has higher priority than `license-judgment.json` :)

Reply:
Maybe the above should be mentioned somewhere as a _guideline to use `package-judgment.json`_.
If not, I am worrying that people will try to add all package names in a PR into `package-judgment.json`.